### DISCUSSION TITLE
Support for China

### DISCUSSION THREAD

**Discussion by NebulaMao** [thatskymod/Sky-CotL-Scripts#81] [Ideas]
It doesn't look like it supports the Chinese version.,Show not installed.,I hope it can be supported. itGame package name (com.netease.sky)

**Comment by madiajijah11** [comment]
Maybe you can try similar method for emulator, here:

1. Find and download an APK editor of your choice.
- Good option for Android: https://maximoff.su/apktool/?lang=en

2. Download the latest Canvas APK, then in your APK editor find where to edit the Package Name of the Canvas APK.
- If you are using the APK Editor mentioned above, tap the Canvas APK and choose Quick Edit.

3. Change the Package Name from git.artdeell.skymodloader to com.netease.sky or similar.
- Optionally you can change the "App Name" to something else to help differentiate the app.

4. Save the changes to the APK and install the new one you have created.

**Reply by NebulaMao** [reply]
Sorry, I still don't understand.I tried to change the package name from （com.netease.sky）to（com.tgc.sky.android）But something went wrong。
![Screenshot_20240505_234348_git artdeell skymodloader](https://github.com/thatskymod/Sky-CotL-Scripts/assets/51261802/b6652929-209a-4551-a6d1-88030340fe62)

**Reply by madiajijah11** [reply]
Sorry, can you give me the screenshot of APKTool when renaming the package? It should be com.netease.sky, I mean renaming it like the package name of the game sky.

**Reply by NebulaMao** [reply]
This is the problem that occurs during startup. I can offer you a Chinese version of the game apk.

This onedrive link[Download](https://yxrsss-my.sharepoint.com/:u:/g/personal/xmccln_yxrsss_onmicrosoft_com/Ecuwciu91ONMts-7WEvohq0Bizb5Fo3FbXjnScrwKUPdaA?e=h9N9Ec)

**Reply by NebulaMao** [reply]
sorry I won't modify the APK。

**Comment by madiajijah11** [comment]
I found some information on the Sky: Children of the Light Wiki. Which is the [Chinese](coco://sendMessage?ext=%7B%22s%24wiki_link%22%3A%22https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FChinese_language%22%7D&msg=Chinese) localization of Sky: Children of the Light. It was published by NetEase, but the game was mostly developed by [thatgamecompany](coco://sendMessage?ext=%7B%22s%24wiki_link%22%3A%22https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FThatgamecompany%22%7D&msg=thatgamecompany). The differences might be related to the Seasons, Special Events, Traveling Spirits, Cosmetics, Timeline, Patch Notes, Key Art, localization and development aspects, which could be causing the canvas to be incompatible with the Chinese version.

Source: [Sky: Children of the Light Wiki](https://sky-children-of-the-light.fandom.com/wiki/NetEase)

**Reply by NebulaMao** [reply]
thanks for you help . I really hope developers can see and adapt.

**Comment by thatskymod** [comment]
As noted by @madiajijah11, the NetEase-published Chinese version of Sky: Children of the Light isn’t just the global version with Chinese translations; it uses separate servers, asset management, event schedules, and patch cycles that differ fundamentally from TGC’s standard release. Manually changing the package name (from `com.netease.sky` to something else) won’t address those deeper architectural differences. Canvas and its mods are tightly coupled to the global edition’s codebase, so supporting the Chinese version would require a complete rewrite—a full-time job requiring a dedicated project or branch covering everything from data calls to file paths, network protocols, and more.

Above all, I understand your situation and truly hope you’ll find a solution that works for you soon!

**Reply by 2859163417** [reply]
Are there any plans to develop a PC version of 'The Sky Mod' for NetEase's Chinese release of Sky: Children of the Light? If so, any estimated release timeline? Thanks!

**Reply by thatskymod** [reply]
Hey there! I’m not affiliated with the ThatSkyMod developer team and don’t officially represent them (despite the name). It’s best to ask directly in their Discord (if that works where you live): https://discord.com/invite/am44RYcXYC — or reach out to XeTrinityz on GitHub. I’m sure they’ll have better answers! 🤗

**Comment by thatskymod** [comment]
Not sure if this helps, but ThatSkyMod (PC version) offers a community-provided Chinese translation as a temporary workaround for those who have to access the global server but would prefer a Chinese UI:

1. https://github.com/yxsj245/ThatSkyMod/tree/master/%E7%95%8C%E9%9D%A2%E6%B1%89%E5%8C%96
2. https://discord.com/channels/1247220592919314522/1276482483617140766/1276482483617140766

Of course, this doesn’t resolve compatibility with the NetEase version, and I understand that access to the global server itself can be a challenge. While not a perfect fix, hopefully, this makes things a bit more accessible.
